rotkehlchen/db/addressbook.py

```diff
@@ -48,45 +49,28 @@ def write_ctx(self, book_type: AddressbookType) -> Generator['DBCursor', None, N
     def get_addressbook_entries(
             self,
             cursor: 'DBCursor',
-            optional_chain_addresses: Optional[list[OptionalChainAddress]] = None,
-    ) -> list[AddressbookEntry]:
+            filter_query: Optional['AddressbookFilterQuery'] = None,
```

Comment:
why make it optional? It complicates the logic and irc in other similar functions we always provide it, even if is empty

Reply:
You mean, I shouldn't provide a default value of None?, or I should not make it optional at all? i.e. users should create an empty filter?